src/components/dropdown/dropdown.js

```diff
@@ -24,7 +24,9 @@ export default class Dropdown extends PureComponent {
     value: PropTypes.any,
     onChange: PropTypes.func,
     color: PropTypes.oneOf(colors),
-    align: PropTypes.oneOf(['right']),
```

Comment:
Can you please keep this prop for now (to avoid breaking this to other people) and add a deprecation warning with this prop it's used)

Reply:
OK no problem; should've thought of that.

Reply:
Would be better if the warning is shown in the doc rather than in the console, since ESLint prohibits `console.warn`. Though, I showed the deprecation warning in both the storybook and in the console as a warning. @couds Thoughts?